apps/meteor/server/models/raw/Rooms.ts

```diff
@@ -1348,7 +1348,7 @@ export class RoomsRaw extends BaseRaw<IRoom> implements IRoomsModel {
 			t: {
 				$in: types,
 			},
-			...(defaultValue ? { default: true } : { default: { $exists: false } }),
```

Comment:
If `default` is always there, maybe the type should not be optional? 👀

Reply:
Actually, it should only be there once the room is set as default. After that, setting it to false should not erase the field so as to not break the `setDefault` endpoints' returns.
Just creating a non-default room won't create this field too, it's not always there.